.pylintrc

```diff
@@ -48,7 +48,7 @@ ignore=CVS
 # ignore-list. The regex matches against paths and can be in Posix or Windows
 # format. Because '\' represents the directory delimiter on Windows systems, it
 # can't be used as an escape character.
-ignore-paths=
+ignore-paths=venv
```

Comment:
As with other comment I think this can be reverted, if pipenv is used with a venv outside the runner folder